scripts/magpietts/codec_extraction.py

```diff
@@ -86,8 +86,9 @@ def __init__(self, model_path):
         self.codec_model.eval()
 
     def forward(self, batch):
-        with torch.no_grad():
-            codes, codes_lengths = self.codec_model.encode(audio=batch["audios"], audio_len=batch["audio_lengths"])
+        with torch.cuda.amp.autocast(enabled=False):
```

Comment:
@paarthneekhara, want to follow up this change. Any reason why we add this context manager? when generating codec codes, we always use default trainer's default precision, which is `32-true`. We don't use mixed precision for this effort.